### DISCUSSION TITLE
Could this be included in windhawk_utils.h?

### DISCUSSION THREAD

**Discussion by Anixx** [ramensoftware/windhawk#170] [Ideas]
This code block is needed in many mods. Can we instead include it in windhawk_utils.h?

```
#ifdef _WIN64
#define CALCON __cdecl
#define SCALCON L" __cdecl "
#else
#define CALCON __thiscall
#define SCALCON L" __thiscall "
#endif
```

Note that the spaces are important here under the quotes...

**Comment by m417z** [comment]
I, personally, never used this kind of block. It's not something universally correct (see [the recent comment](https://github.com/ramensoftware/windhawk-mods/pull/549/files#r1510658708) for example) and I'd rather not having it included.

**Reply by Anixx** [reply]
Can you please tell, what should be the best practice?

**Reply by m417z** [reply]
I'm not aware of a universal best practice or convention. Most or all of my explorer.exe mods are 64-bit only, so there was no need for that.
